examples/simple-udp/main.go

```diff
@@ -24,9 +24,10 @@ import (
 	"strings"
 	"time"
 
```

Comment:
Super mega nit: would love to see this blank line gone :)

Reply:
goimport force it, I guess for separetion of standard vs community

Reply:
I mean the one between "time" and "strconv"

Reply:
I'll give a try

Reply:
I've noticed goimports will leave in random blank lines in places, and they just need some manual cleanup.

Reply:
right ! I'm using goreturns though

Reply:
I'm willing to bet if you remove the blank line between it will stay removed. I'd put $10 on it using the same code as goimports to sort the import statements.

Reply:
you were right